apps/condo/domains/common/hooks/usePrevious.tsx

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useRef } from 'react'
+
+/**
+ * Monitoring some variable value
+ * After it's change useEffect will update ref
+ * But before that will be returned previous value
+ * @param value to monitor
+ */
+export function usePrevious<T> (value: T): T {
+    const ref = useRef<T>()
```

Comment:
First execution of this function will return `undefined` result.
Maybe to initialize it first?

```suggestion
    const ref = useRef<T>(value)
```

Reply:
We don't want to do it here...Since previous of first value is nothing 😄